ajax/libs/darktooltip/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "darktooltip",
+  "filename": "jquery.darktooltip.min.js",
+  "version": "0.4.0",
+  "description": "A simple customizable tooltip with confirm option and effects",
+  "keywords": [
+    "tooltip",
+    "confirm"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/rubentd/darktooltip.git"
+  },
+  "homepage": "http://rubentd.com/darktooltip",
+  "author": "Ruben Torres",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/rubentd/darktooltip.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "*darktooltip.@(js|css)",
+          "modal-bg.png"
```

Comment:
@hare1039 We should try to apply rules to exclude files we don't need under `dist` folder.